tests/c_cpp/cppcheck/main.c

```diff
@@ -0,0 +1,9 @@
+// Code taken from the CppCheck manual (page 4)
+// http://cppcheck.sourceforge.net/manual.pdf
+
+int main()
+{
+    char a[10];
+    a[10] = 0;
```

Comment:
Codacy found an issue: [Variable 'a[10]' is assigned a value that is never used.](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6081579)